### DISCUSSION TITLE
Unable to use SmartTables

### DISCUSSION THREAD

**Discussion by 0xblockbard** [aptos-labs/aptos-developer-discussions#392] [Questions] [answered]
Hello all, have been unable to use smart tables, even on very simple use cases, does anyone know what's wrong?

Have been following the docs here: https://aptos.dev/en/build/smart-contracts/smart-table

Sample code:
use aptos_std::smart_table::{Self, SmartTable};

let removed_value = SmartTable::remove(&mut table, 1);
   │                             ^^^^^^^^^^ Unbound module or type alias 'SmartTable'

**Comment by gregnazario** [comment, marked as answer]
All functions have to use the module name, instead of the type name.  Try

```move
module 0x42::example {
  use aptos_std::smart_table::{self, SmartTable};

  fun do_something(){
    let key:u8 = 1;
    let table = smart_table::new<u8, u8>();
    let removed_value = smart_table::remove(&mut table, key);
  }
}
```

Looks like the docs are wrong and we'll fix it

**Reply by 0xblockbard** [reply]
Great! I got it working now!
